### PR TITLE
Ndj uploads take 2

### PR DESCRIPTION
**Pull request by NicholasJacques**

### CODE REVIEW

**Comment by NicholasJacques** [comment]
@jmejia here It is, I think

File: app/controllers/bid_controller.rb [hunk 1]
```diff
@@ -5,6 +5,9 @@ def create
     @bid.project = Project.find(params[:project_id])
 
     if @bid.save
+      if params[:bid][:attachments_attributes]
+        @bid.attachments.create(upload: params[:bid][:attachments_attributes]["0"][:upload])
```
Comment (jmejia) [inline app/controllers/bid_controller.rb:9]: Clever hack 😉

File: app/models/attachment.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class Attachment < ApplicationRecord
+  belongs_to :attachable, polymorphic: true
```
Comment (jmejia) [inline app/models/attachment.rb:2]: Models look really good 👍

**Review by jmejia — COMMENTED** [review]
Nice job getting this to work. Some room for refactoring but you did the most important things. Nice job.

**Pull request merged**